pydantic/schema.py

```diff
@@ -6,7 +6,8 @@
 from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Sequence, Set, Tuple, Type, Union, cast
 from uuid import UUID
 
-from . import main
+import pydantic
```

Comment:
Had to modify how `pydantic.main` was accessed due to the way cython handles imports (was causing a cyclic import issue).